### ISSUE TITLE
Gallery on a post isn't working

### ISSUE THREAD

**Issue by MangopearUK** [MangopearUK/European-Boating-Association--Theme#406]
Link: https://eba.eu.com/2015/08/donnau-danube-apps-fur-smartphones/

Popups aren't working.

## A11y failure too:

Links do not have a discernible name

Link text (and alternate text for images, when used as links) that is discernible, unique, and focusable improves the navigation experience for screen reader users. Learn more.

Failing Elements
--
a
a
a
a
a
a
a
a
a
a

![2019-09-17-15-48-eba eu com (1)](https://user-images.githubusercontent.com/3357509/65052566-8ef5b300-d962-11e9-8098-daa4abd14ba4.png)
